src/mon/Monitor.cc

```diff
@@ -2392,12 +2495,28 @@ health_status_t Monitor::get_health(list<string>& status,
   if (f)
     f->close_section();
 
-  if (f)
+  fss.clear();
+  fss << std::endl;
+  scrubbed_pgs_info_t scrubbed_pgs_info = (detailbl == nullptr ?
+					   scrubbed_pgs_info_t::COUNT :
+					   scrubbed_pgs_info_t::DETAILED);
+
+  print_unscrubbed_pgs(pgmon()->pg_map.pg_stat,
+		       fss,
+		       f,
+		       cct,
+		       scrubbed_pgs_info);
+
+  if (f) {
     f->close_section();
+  } else {
+    status.push_back(fss.str());
```

Comment:
Should this be guarded by a conditional?  We don't want to push an empty string if there are no unscrubbed pgs.